src/native/libs/CMakeLists.txt

```diff
@@ -4,8 +4,13 @@ include(CheckCCompilerFlag)
 
 project(CoreFX C)
 
-include(../../../eng/native/configurepaths.cmake)
-include(${CLR_ENG_NATIVE_DIR}/configurecompiler.cmake)
+if ("${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL "${CMAKE_SOURCE_DIR}")
```

Comment:
Why is this condition needed now and wasn't needed before? Is it fixing some pre-existing warning?

Reply:
We used to have an include guard in `configurecompiler.cmake` as it could get included twice from here when the libs CMake is included in the CoreCLR CMake tree. Adding this condition allows us to remove the include guard.